apps/ledger-live-mobile/src/screens/ReceiveFunds/03-Confirmation.tsx

```diff
@@ -255,8 +277,15 @@ function ReceiveConfirmationInner({ navigation, route, account, parentAccount }:
         : null;
   }
 
+  const isNotATokenAccount = (account: TokenAccount | Account): account is Account => {
```

Comment:
You can do just `account.type === "Account"` imo

Reply:
Indeed ^^